src/frontend/pages/chat/message/message.js

```diff
@@ -91,6 +91,7 @@ export class Message extends LitElement {
 
         <il-message-menu-popover
           .chatRef=${this.chatRef}
+          .snackbarRef=${this.snackbarRef}
```

Comment:
Mi sembra che di solito quando c'è un componente child che fa qualcosa in un componente parent si utilizzino gli eventi.